packages/vite-plugin/tests/out/vite-content-script-css-imports-2/__snapshots__/build.test.ts.snap

```diff
@@ -73,48 +71,14 @@ exports[`build fs output > assets/index.hash3.css 1`] = `
 "
 `;
 
-exports[`build fs output > assets/index.ts.hash4.js 1`] = `
-"const styles = \\"\\";
-console.log(\\"content2\\");
-"
-`;
-
-exports[`build fs output > assets/index.ts.hash5.js 1`] = `
-"const styles = \\"\\";
+exports[`build fs output > assets/index.ts.hash0.js 1`] = `
+"(function(){const styles = \\"\\";
 console.log(\\"content1\\");
-"
-`;
-
-exports[`build fs output > assets/index.ts-loader.hash0.js 1`] = `
-"(function () {
-  'use strict';
-
-  const injectTime = performance.now();
-  (async () => {
-    const { onExecute } = await import(
-      /* @vite-ignore */
-      chrome.runtime.getURL(\\"assets/index.ts.hash5.js\\")
-    );
-    onExecute?.({ perf: { injectTime, loadTime: performance.now() - injectTime } });
-  })().catch(console.error);
-
-})();
-"
+})()"
 `;
 
-exports[`build fs output > assets/index.ts-loader.hash2.js 1`] = `
-"(function () {
-  'use strict';
-
-  const injectTime = performance.now();
-  (async () => {
-    const { onExecute } = await import(
-      /* @vite-ignore */
-      chrome.runtime.getURL(\\"assets/index.ts.hash4.js\\")
-    );
-    onExecute?.({ perf: { injectTime, loadTime: performance.now() - injectTime } });
```

Comment:
- do we need some kind of performance metrics for this scenario?

Reply:
good catch - indeed, looks like `onExecute` is an optional callback that content scripts may export which the crxjs loader should call: https://github.com/crxjs/chrome-extension-tools/pull/689

given that in order to call a content script directly we require it to have no imports/exports, it is by definition not needed to try to call such callback in case of "direct" injection. I will add a comment clarifying that for future reference 👍

Reply:
https://github.com/crxjs/chrome-extension-tools/pull/997/commits/f6755ccbcb6b7a4223f75f97687cff365e128a3d